src/CoralBenchmark.cc

```diff
@@ -5,20 +5,41 @@
 #include "utilsMpi.hh"
 #include "MC_Processor_Info.hh"
 
+void BalanceRatioTest( MonteCarlo* monteCarlo, Parameters &params );
+void MissingParticleTest( MonteCarlo* monteCarlo );
+void FluenceTest( MonteCarlo* monteCarlo );
 
-#ifdef CORAL_2_BENCHMARK
-void scalarFluxTest( MonteCarlo* monteCarlo );
-#endif
-
-#ifdef CORAL_2_BENCHMARK
 void coralBenchmarkCorrectness( MonteCarlo* monteCarlo, Parameters &params )
 {
+    if( params.simulationParams.coralBenchmark )
```

Comment:
Don't make me think about multiple conditionals.

if (!benchmark) return;

And write a comment explaining that this is only relevant to the Coral Benchmark problem